src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
+import getToken from '../lib/tokens';
+import { Redirect } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 
 const Home = () => {
-  return <h1>Home</h1>;
+  // const [hasToken, setHasToken] = useState(false);
+
+  let history = useHistory();
+
+  const Token = function generateToken() {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    history.push('/list-view');
+
+    console.log(token);
+  };
+
+  // function fetchToken() {
+  //   localStorage.getItem('token');
+  //   history.push('/list-view');
+  // }
+
+  const existingToken = localStorage.getItem('token');
+
+  return (
+    <div>
+      <button onClick={Token}>Create a new list</button>
+      {existingToken ? <Redirect from="/" to="/list-view" /> : ''}
+      {
+        {
+          /* <input> below will be used at a later date  */
+        }
```

Comment:
We're getting this error in the preview build and locally because of this empty object. If you remove these two curly braces on line 30 & 33, it will build properly. 👍 
> [Objects are not valid as a React child (found: object with keys {}). If you meant to render a collection of children, use an array instead.](https://reactjs.org/docs/error-decoder.html/?invariant=31&args[]=object%20with%20keys%20%7B%7D)

Reply:
Awesome !!!!  its working properly on home page now

Reply:
Thanks both for finding and solving this error!